build/test-runtime.js

```diff
@@ -17,8 +17,11 @@ module.exports.TestRuntime = function({
     const base = '.fusion/dist/test';
     const server = path.resolve(dir, base, 'server/server-main.js');
     const client = path.resolve(dir, base, 'client/client-main.js');
-    let command = require.resolve('unitest/bin/cli.js');
+    // let command = require.resolve('unitest/bin/cli.js');
```

Comment:
Is it intentional to leave this commented?

Reply:
good catch - updating